graf2d/gpadv7/v7/inc/ROOT/RAttrLine.hxx

```diff
@@ -42,6 +42,48 @@ public:
    int GetStyle() const { return Get<int>("style"); }
 };
 
+
+class RAttrLineNew : public RAttributesVisitor {
+
+   auto &getDefaults()
+   {
+      static RDrawableAttributes::Map_t dflts;
+      if (dflts.empty()) {
```

Comment:
This is not thread safe.

Reply:
Yes, I know. I didnot find compact way for current implementation with `std::unordered_map<std::string, std::unique_ptr<Value_t>>`. Problem that `unique_ptr` does not like copy constructor. But after I merge #4361, I can make subclass and provide convenience methods to initialize such object.